lib/beaker-http/http.rb

```diff
@@ -33,31 +37,18 @@ def create_default_connection
         end
       end
 
-      def set_url_prefix(connection)
-        connection.url_prefix.host = @host.hostname
-        nil
-      end
-
-      def create_default_connection_with_beaker_host
-        connection = create_default_connection
-        set_url_prefix(connection)
-        connection
-      end
-
       # If you would like to run tests that expect 400 or even 500 responses,
       # apply this method to remove the <tt>:raise_error</tt> middleware.
-      def remove_error_checking(connection)
+      def remove_error_checking
         connection.builder.delete(Faraday::Response::RaiseError)
         nil
       end
 
-
-
       # Use this method if you are connecting as a user to the system; it will
       # provide the correct SSL context but not provide authentication.
       def configure_cacert_only_with_puppet
```

Comment:
we might want to make this more general, but i'm fine with it for now. 
e.g.: this might want to move to a puppet library.  and users in general can just set the ssl stuff themselves for general Http Connections.